server/sandbox_run_linux.go

```diff
@@ -42,6 +42,12 @@ import (
 // DefaultUserNSSize is the default size for the user namespace created
 const DefaultUserNSSize = 65536
 
+const (
+	kubeletPodsRootDir    = "/var/lib/kubelet/pods"
+	kubeletPodLogsRootDir = "/var/log/pods"
+	kubeletEmptyDirLogDir = "kubernetes.io~empty-dir"
```

Comment:
I find myself wishing we had public variables in k8s we could reference here, but I'm not seeing any...